commit c94992179e99f4088f3a1cb8e304a9fb6035170d
Author: John Vogel <jvogel4@stny.rr.com>
Date:   Sat Mar 17 12:33:24 2012 -0400

    Update manpage to current version. (Fix bug 5444)

diff --git a/doc/openbox.1.in b/doc/openbox.1.in
index d508e8c..bd3747f 100644
--- a/doc/openbox.1.in
+++ b/doc/openbox.1.in
@@ -58,11 +58,14 @@ to ~/.config/openbox and edit it to your liking.
 .PP 
 These are the possible options that \fBopenbox\fR accepts: 
 .IP "\fB\-\-help\fP" 10 
-Show a summary of the options available. 
+Show a summary of the command line options available 
+and exit. 
 .IP "\fB\-\-version\fP" 10 
-Show the version of the program. 
+Show the version of the program and exit. 
 .IP "\fB\-\-replace\fP" 10 
 Replace the currently running window manager. 
+.IP "\fB\-\-config-file FILE\fP" 10 
+Specify the path to the config file to use. 
 .IP "\fB\-\-reconfigure\fP" 10 
 If Openbox is already running on the display, tell it to 
 reload its configuration. 
@@ -70,14 +73,20 @@ reload its configuration.
 If Openbox is already running on the display, tell it to 
 restart. This is useful if you have upgraded Openbox and don't 
 want to restart X. 
+.IP "\fB\-\-exit\fP" 10 
+Exit Openbox. 
 .IP "\fB\-\-sm-disable\fP" 10 
 Do not connect to the session manager. 
 .IP "\fB\-\-sync\fP" 10 
 Run in synchronous mode (for debugging). 
+.IP "\fB\-\-startup CMD\fP" 10 
+Run CMD after starting. 
 .IP "\fB\-\-debug\fP" 10 
 Display debugging output. 
 .IP "\fB\-\-debug-focus\fP" 10 
 Display debugging output for focus handling. 
+.IP "\fB\-\-debug-session\fP" 10 
+Display debugging output for session management. 
 .IP "\fB\-\-debug-xinerama\fP" 10 
 Split the display into two fake xinerama regions, if 
 xinerama is not already enabled. This is for debugging 
@@ -92,4 +101,4 @@ The program's full documentation is available on the website:
 .PP 
 Please report bugs to: \fBhttp://bugzilla.icculus.org/ 
 \fP 
-.\" created by instant / docbook-to-man, Wed 06 Jan 2010, 13:40 
+.\" created by instant / docbook-to-man, Sun 30 Sep 2012, 17:58 
diff --git a/doc/openbox.1.sgml b/doc/openbox.1.sgml
index 275c324..281f2a0 100644
--- a/doc/openbox.1.sgml
+++ b/doc/openbox.1.sgml
@@ -106,13 +106,14 @@ manpage.1: manpage.sgml
       <varlistentry>
         <term><option>--help</option></term>
         <listitem>
-          <para>Show a summary of the options available.</para>
+          <para>Show a summary of the command line options available
+            and exit.</para>
         </listitem>
       </varlistentry>
       <varlistentry>
         <term><option>--version</option></term>
         <listitem>
-          <para>Show the version of the program.</para>
+          <para>Show the version of the program and exit.</para>
         </listitem>
       </varlistentry>
       <varlistentry>
@@ -122,6 +123,12 @@ manpage.1: manpage.sgml
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><option>--config-file FILE</option></term>
+        <listitem>
+          <para>Specify the path to the config file to use.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
         <term><option>--reconfigure</option></term>
         <listitem>
           <para>If Openbox is already running on the display, tell it to
@@ -137,6 +144,12 @@ manpage.1: manpage.sgml
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><option>--exit</option></term>
+        <listitem>
+          <para>Exit Openbox.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
         <term><option>--sm-disable</option></term>
         <listitem>
           <para>Do not connect to the session manager.</para>
@@ -149,6 +162,12 @@ manpage.1: manpage.sgml
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><option>--startup CMD</option></term>
+        <listitem>
+          <para>Run CMD after starting.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
         <term><option>--debug</option></term>
         <listitem>
           <para>Display debugging output.</para>
@@ -161,6 +180,12 @@ manpage.1: manpage.sgml
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><option>--debug-session</option></term>
+        <listitem>
+          <para>Display debugging output for session management.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
         <term><option>--debug-xinerama</option></term>
         <listitem>
           <para>Split the display into two fake xinerama regions, if
--- a/doc/openbox-session.1.in
+++ b/doc/openbox-session.1.in
@@ -12,8 +12,8 @@ to save your state from one log in to th
 arguments. 
 .PP 
 On log in, \fBopenbox-session\fR will run the 
-~/.config/openbox/autostart.sh script if it exists, and will run the 
-system-wide script @configdir@/openbox/autostart.sh otherwise. You may 
+~/.config/openbox/autostart script if it exists, and will run the 
+system-wide script @configdir@/openbox/autostart otherwise. You may 
 place anything you want to run automatically in those files, for example: 
  
 .PP 
@@ -27,7 +27,7 @@ gnome-settings-daemon &
 .PP 
 Make sure that each line is followed by a "&" or else the script will 
 stop there and further commands will not be executed. You can use the 
-@configdir@/openbox/autostart.sh file as an example for creating your 
+@configdir@/openbox/autostart file as an example for creating your 
 own. 
 .SH "SEE ALSO" 
 .PP 
--- a/doc/openbox-session.1.sgml
+++ b/doc/openbox-session.1.sgml
@@ -44,8 +44,8 @@ manpage.1: manpage.sgml
       arguments.</para>
 
     <para>On log in, <command>&dhpackage;</command> will run the
-      ~/.config/openbox/autostart.sh script if it exists, and will run the
-      system-wide script @configdir@/openbox/autostart.sh otherwise. You may
+      ~/.config/openbox/autostart script if it exists, and will run the
+      system-wide script @configdir@/openbox/autostart otherwise. You may
       place anything you want to run automatically in those files, for example:
       </para>
 
@@ -55,7 +55,7 @@ manpage.1: manpage.sgml
 
     <para>Make sure that each line is followed by a "&" or else the script will
       stop there and further commands will not be executed. You can use the
-      @configdir@/openbox/autostart.sh file as an example for creating your
+      @configdir@/openbox/autostart file as an example for creating your
       own.</para>
 
   </refsect1>

