commit 5aad740a527e4aa6b1a96dadecbb6d6a29cc7e7c
Author: Dana Jansens <danakj@orodu.net>
Date:   Mon Oct 17 15:48:31 2011 -0400

    Quiet build outbut on by default

diff --git a/README.GIT b/README.GIT
index dfed9c4..a6ad99b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,8 +1,10 @@
 AC_PREREQ([2.54])
 AC_INIT([openbox], [3.5.0], [http://bugzilla.icculus.org])
-AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([openbox/openbox.c])
 
+AM_INIT_AUTOMAKE([foreign])
+AM_SILENT_RULES([yes])
+
 OB_VERSION=$PACKAGE_VERSION
 AC_SUBST(OB_VERSION)
 
