From b0d0a5e2b875c5258ce6b4591b0bdcc6b48b47b3 Mon Sep 17 00:00:00 2001
From: Zsolt Salamon <github@sa4zet.win>
Date: Sun, 12 Jul 2020 22:08:04 +0200
Subject: [PATCH] Allow 256x256 window icon size for HiDPI displays

---
 openbox/config.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/openbox/config.c b/openbox/config.c
index dad5d1b..78b567f 100644
--- a/openbox/config.c
+++ b/openbox/config.c
@@ -718,8 +718,8 @@ static void parse_theme(xmlNodePtr node,
         config_theme_window_list_icon_size = obt_xml_node_int(n);
         if (config_theme_window_list_icon_size < 16)
             config_theme_window_list_icon_size = 16;
-        else if (config_theme_window_list_icon_size > 96)
-            config_theme_window_list_icon_size = 96;
+        else if (config_theme_window_list_icon_size > 256)
+            config_theme_window_list_icon_size = 256;
     }
     if ((n = obt_xml_find_node(node, "roundCorners")))
         config_theme_roundcorners = obt_xml_node_bool(n);
