Description: Change libobt abi
Author: Mateusz Łukasik <mati75@linuxmint.pl>
Bug-Debian: https://bugs.debian.org/788520

--- a/configure.ac
+++ b/configure.ac
@@ -38,8 +38,8 @@ RR_VERSION=$RR_MAJOR_VERSION.$RR_MINOR_V
 OBT_MAJOR_VERSION=3
 OBT_MINOR_VERSION=6
 OBT_MICRO_VERSION=4
-OBT_INTERFACE_AGE=2
-OBT_BINARY_AGE=2
+OBT_INTERFACE_AGE=3
+OBT_BINARY_AGE=1
 OBT_VERSION=$OBT_MAJOR_VERSION.$OBT_MINOR_VERSION
 
 AC_SUBST(RR_MAJOR_VERSION)
