From 09cee1858488801531baaad9608432f625e67606 Mon Sep 17 00:00:00 2001
From: Zsolt Salamon <github@sa4zet.win>
Date: Sat, 18 Jul 2020 21:02:24 +0200
Subject: [PATCH] Configurable up and down arrows in themes

---
 obrender/theme.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/obrender/theme.c b/obrender/theme.c
index 2a4f6e1..fad69cd 100644
--- a/obrender/theme.c
+++ b/obrender/theme.c
@@ -528,10 +528,12 @@ RrTheme* RrThemeNew(const RrInstance *inst, const gchar *name,
     }
 
     /* up and down arrows */
+    if (!read_mask(inst, path, "down_arrow.xbm", &theme->down_arrow_mask))
     {
         guchar data[] = { 0xfe, 0x00, 0x7c, 0x00, 0x38, 0x00, 0x10, 0x00 };
         theme->down_arrow_mask = RrPixmapMaskNew(inst, 9, 4, (gchar*)data);
     }
+    if (!read_mask(inst, path, "up_arrow.xbm", &theme->up_arrow_mask))
     {
         guchar data[] = { 0x10, 0x00, 0x38, 0x00, 0x7c, 0x00, 0xfe, 0x00 };
         theme->up_arrow_mask = RrPixmapMaskNew(inst, 9, 4, (gchar*)data);
-- 
2.28.0.rc1

